python_modules/libraries/dagster-dbt/dagster_dbt_tests/core/test_asset_checks.py

```diff
@@ -575,6 +579,126 @@ def my_dbt_assets(context: AssetExecutionContext, dbt: DbtCliResource):
+                    "dagster_dbt/failed_row_count": 0,
+                },
+            ),
+            AssetCheckEvaluation(
+                passed=True,
+                asset_key=AssetKey(["customers"]),
+                check_name="not_null_customers_customer_id",
+                metadata={
+                    "unique_id": (
+                        "test.test_dagster_asset_checks.not_null_customers_customer_id.5c9bf9911d"
+                    ),
+                    "invocation_id": invocation_id,
+                    "status": "pass",
+                    "dagster_dbt/failed_row_count": 0,
+                },
+            ),
+            AssetCheckEvaluation(
+                passed=False,
+                asset_key=AssetKey(["fail_tests_model"]),
+                check_name="unique_fail_tests_model_id",
+                severity=AssetCheckSeverity.WARN,
+                metadata={
+                    "unique_id": (
+                        "test.test_dagster_asset_checks.unique_fail_tests_model_id.1619308eb1"
+                    ),
+                    "invocation_id": invocation_id,
+                    "status": "warn",
+                    "dagster_dbt/failed_row_count": 1,
+                },
+            ),
+            AssetCheckEvaluation(
+                passed=False,
+                asset_key=AssetKey(["fail_tests_model"]),
+                check_name="accepted_values_fail_tests_model_first_name__foo__bar__baz",
+                severity=AssetCheckSeverity.ERROR,
+                metadata={
+                    "unique_id": (
+                        "test.test_dagster_asset_checks.accepted_values_fail_tests_model_first_name__foo__bar__baz.5f958cf018"
+                    ),
+                    "invocation_id": invocation_id,
+                    "status": "fail",
+                    "dagster_dbt/failed_row_count": 4,
+                },
+            ),
+        ]
+
+        # filter these out for comparison
+        non_deterministic_metadata_keys = ["Execution Duration"]
+        check_events_without_non_deterministic_metadata = {}
+        for event in events:
+            if isinstance(event, AssetCheckEvaluation):
+                check_events_without_non_deterministic_metadata[
+                    event.asset_key, event.check_name
+                ] = event._replace(
+                    metadata={
+                        k: v
+                        for k, v in event.metadata.items()
+                        if k not in non_deterministic_metadata_keys
+                    }
+                )
+
+        for expected_asset_check_evaluation in expected_results:
+            assert (
+                check_events_without_non_deterministic_metadata[
+                    expected_asset_check_evaluation.asset_key,
+                    expected_asset_check_evaluation.check_name,
+                ]
+                == expected_asset_check_evaluation
+            )
+
+        yield from events
+
+        @job
```

Comment:
@maximearmstrong i think this needs to be unindented in order for the test to actually execute

Reply:
Good catch - fixed here https://github.com/dagster-io/dagster/pull/28551